### PR TITLE
Fix kornia 0.8.3 pyramid padding compatibility

### PR DESCRIPTION
**Pull request by ufotone**
This removes the import of `pad` from `kornia.geometry.transform.pyramid`, which is no longer available in kornia 0.8.3.

Padding now uses `torch.nn.functional.pad`, preserving the existing behavior while allowing ComfyUI-LTXVideo to import successfully with kornia 0.8.3.

Also relaxes the kornia requirement to `kornia>=0.8.2,<0.9`.

Tested locally with:
- ComfyUI 0.28.0
- PyTorch 2.13.0+cu130
- kornia 0.8.3

ComfyUI-LTXVideo loads without `IMPORT FAILED`.

### CODE REVIEW

**Comment by JavierMedel** [comment]
I can confirm this issue on the latest `master` (`3b9c5cd`) with Kornia `0.8.3`.

The failure is caused by importing `pad` from `kornia.geometry.transform.pyramid`, where it is no longer available.

I tested replacing the Kornia `pad` calls with `torch.nn.functional.pad`:

```python
image = F.pad(image, (0, pad_right, 0, pad_down), mode="reflect")
images = F.pad(images, padding, mode=border_type)
```

The file passes `py_compile`, the required Kornia imports work, and ComfyUI loads the LTXVideo node without the `ImportError`.

I also have this fix committed locally (`b6c4087`) if it helps with testing.
